angular/src/app/shared/data-table/data-table.component.ts

```diff
@@ -35,19 +34,14 @@ export class DataTableComponent implements OnDestroy {
 
   @Output()
   readonly selectedRowsChanged: EventEmitter<any | any[]> = new EventEmitter<any | any[]>();
+  @Output()
+  readonly rowClicked: EventEmitter<any> = new EventEmitter<any>();
```

Comment:
This is actually not right:
1. It is not **row clicked** event it is **cell clicked**, the row click event is **selectedRowChanged**
2. Instead of creating new output you can just reuse the existing `selectedRowChanged`

Reply:
I see what you mean, but selectedRowChanged is not always the same as a rowClicked event. For example if you use the arrow keys to change row selection, a row was never clicked, but the a selectedRowChanged is emitted. I did originally reuse selectedRowChanged (open the details every time the event is emitted), but the issue is that it would open the details when navigating using the arrow keys, which I think is odd behaviour. But if that is what is intended behaviour, then we can do that.

Reply:
I can also rename rowClicked to cellClicked to be more accurately named

Reply:
I would say that is the intended behaviour. It is also the current behaviour in deployed COVID-19 app.